global_base_frame: &global_base_frame map
global_child_frame: &global_child_frame lidar

## Point cloud
pointcloud_topic: /lslidar_point_cloud
# pointcloud_topic: /synced_cloud

## Front end
lidar_odom_topic: 
    name: /laser_odom
    base_frame: *global_base_frame
    child_frame: *global_child_frame

## Back end
loop_topic: /loop_pose
gnss_topic: /synced_gnss
optimized_odom_topic:
    name: /optimized_pose
    base_frame: *global_base_frame
    child_frame: *global_child_frame
path_topic:
    name: /path
    base_frame: *global_base_frame

use_gnss: false
use_loop_close: false

## Viewer
map_topic:
    name: /back_end_map
    base_frame: *global_base_frame
scan_topic:
    name: /back_end_scan
    base_frame: *global_base_frame
